diff --git a/src/FlysystemBridge.php b/src/FlysystemBridge.php
index 998c37d..c9ea2a4 100644
--- a/src/FlysystemBridge.php
+++ b/src/FlysystemBridge.php
@@ -128,4 +128,12 @@ class FlysystemBridge extends FlysystemStreamWrapper implements StreamWrapperInt
     return \Drupal::service('flysystem_factory');
   }

+  /**
+   * {@inheritdoc}
+   */
+  public function __set(string $name, mixed $value): void {
+
+  }
+
+
 }
