diff --git a/matomo.install b/matomo.install
index 021b86b..c557546 100644
--- a/matomo.install
+++ b/matomo.install
@@ -15,8 +15,9 @@ use Drupal\user\Entity\Role;
  */
 function matomo_install(): void {
   // Make the default install more user and GDPR friendly.
-  $role = Role::load('authenticated');
-  $role->grantPermission('opt-in or out of matomo tracking');
+  user_role_grant_permissions(AccountInterface::AUTHENTICATED_ROLE, [
+	  'opt-in or out of matomo tracking',
+  ]);
   $success = $role->save();
   if ($success) {
     $messenger = \Drupal::messenger();
